sample_consensus/src/sac_model_stick.cpp

```diff
@@ -36,7 +36,6 @@
  *
  */
 
-#include <pcl/pcl_config.h>
```

Comment:
This needs to stay. PCL_NO_PRECOMPILE is defined in pcl_config

Reply:
Ok, I added it back. That header is not needed in pcd_grabber.cpp and image_grabber.cpp though, is it?

Reply:
I looked at those files and they don't seem to be directly relying on any of the macros pcl_config defines.

If there are files which are transitively relying on pcl_config.h, problems will appear but there's nothing we can do there besides "taking it to the face".